### PR TITLE
Update JobSearch page with Brazil location filter

### PR DESCRIPTION
**Pull request by edinhoo**

### CODE REVIEW

File: src/pages/job-search.tsx [hunk 1]
```diff
@@ -181,14 +197,31 @@ const JobSearch: FC = () => {
...
         currentLocationOptions.push(loc ?? initialLocation)
       })
     }
 
+    const filteredBrazilPostings = filteredPostings.filter(
+      (posting: any) => brazilLocations.includes(posting.location)
+    );
+    const filteredBrazilPostingsCount = filteredBrazilPostings && filteredBrazilPostings.length
+      ? filteredBrazilPostings.length
+      : 0
```
Comment (lucis) [inline src/pages/job-search.tsx:217]:
```suggestion
    const filteredBrazilPostingsCount = filteredBrazilPostings?.length ?? 0
```

File: src/pages/job-search.tsx [hunk 2]
```diff
@@ -181,14 +197,31 @@ const JobSearch: FC = () => {
 
     if (initialLocationAgregate.length) {
       initialLocationAgregate.forEach(initialLocation => {
-        if (initialLocation.value === 'undefined') return
+        if (
+          initialLocation.value === 'undefined' ||
+          initialLocation.value === 'Brazil'
+        ) return
```
Comment (lucis) [inline src/pages/job-search.tsx:203]:
```suggestion
        const BRAZIL = 'Brazil'
        // ... replace it elsewhere
        const IGNORED_LOCATIONS = ['undefined', 'Brazil']
        if (IGNORED_LOCATIONS.includes(initialLocation.value)) return
```

**Review by lucis — APPROVED** [review]
Nitpicks...

File: src/pages/job-search.tsx (hunk 2 above)
Comment (edinhoo) [reply src/pages/job-search.tsx:203]: Great. I added `IGNORED_LOCATIONS` and `BRAZIL_LOCATIONS` as constants at the beginning of the code.

Now it's really easier to understand and edit.

**Comment by lucis** [comment]
Merging because it's failing due an intermittent problem

**Pull request merged**